app/assets/javascripts/discourse/app/templates/components/bookmark-list.hbs

```diff
@@ -2,66 +2,59 @@
   {{#load-more selector=".bookmark-list .bookmark-list-item" action=loadMore}}
     <table class="topic-list bookmark-list">
       <thead>
-        {{#if site.mobileView}}
-          <th>&nbsp;</th>
-          <th>{{i18n "topic.title"}}</th>
-          <th>&nbsp;</th>
-        {{else}}
+        {{#unless site.mobileView}}
           <th>{{i18n "topic.title"}}</th>
           <th>&nbsp;</th>
           <th class="post-metadata">{{i18n "post.bookmarks.updated"}}</th>
           <th class="post-metadata">{{i18n "activity"}}</th>
           <th>&nbsp;</th>
-        {{/if}}
+        {{/unless}}
       </thead>
       <tbody>
         {{#each content as |bookmark|}}
           <tr class="topic-list-item bookmark-list-item">
-            {{#if site.mobileView}}
-              <td>
-                {{#if bookmark.post_user_avatar_template}}
-                  <a href={{bookmark.postUser.path}} data-user-card={{bookmark.post_user_username}}>
-                    {{avatar bookmark.postUser avatarTemplatePath="avatar_template" usernamePath="username" namePath="name" imageSize="small"}}
-                  </a>
-                {{/if}}
-              </td>
-            {{/if}}
             <td class="main-link" role="rowheader">
               <span class="link-top-line">
                 <div class="bookmark-metadata">
-                  {{#if bookmark.name}}
+                  {{#if bookmark.reminder_at}}
                     <span class="bookmark-metadata-item">
-                      {{d-icon "info-circle"}}{{bookmark.name}}
+                      {{d-icon "far-clock"}}{{bookmark.formattedReminder}}
                     </span>
                   {{/if}}
-                  {{#if bookmark.reminder_at}}
+                  {{#if bookmark.name}}
                     <span class="bookmark-metadata-item">
-                      {{d-icon "far-clock"}}{{bookmark.formattedReminder}}
+                      {{d-icon "info-circle"}}<span>{{bookmark.name}}</span>
                     </span>
                   {{/if}}
                 </div>
-
                 <div class="bookmark-status-with-link">
                   {{#if bookmark.pinned}}
                     {{d-icon "thumbtack" class="bookmark-pinned"}}
                   {{/if}}
-                  {{topic-status topic=bookmark.topicStatus}}
+                  {{~topic-status topic=bookmark.topicStatus~}}
                   {{topic-link bookmark.topicLink}}
                 </div>
               </span>
-              {{#if bookmark.excerpt}}
-                {{!-- template-lint-disable  --}}
-                <p class="post-excerpt" {{on "click" this.screenExcerptForExternalLink}}>{{html-safe bookmark.excerpt}}</p>
-              {{/if}}
               <div class="link-bottom-line">
                 {{category-link bookmark.category}}
                 {{discourse-tags bookmark mode="list" tagsForUser=tagsForUser}}
               </div>
+              {{#if bookmark.excerpt}}
```

Comment:
We now have an `(and` helper so you can do this:

`{{#if (and bookmark.excerpt site.mobileView bookmark.post_user_avatar_template)}}`